tiledb/api/src/stats.rs

```diff
@@ -0,0 +1,70 @@
+use crate::error::Error;
+use crate::Result as TileDBResult;
+
+pub fn stats_enable() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_enable()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+       Ok(())
+    } else {
+        Err(Error::from("TileDB stats were not successfully enabled."))
+    }
+}
+
+pub fn stats_disable() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_disable()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+        Ok(())
+    } else {
+        Err(Error::from("TileDB stats were not successfully disabled."))
+    }
+}
+
+pub fn stats_reset() -> TileDBResult<()> {
+    let c_ret : i32 = unsafe {
+        ffi::tiledb_stats_reset()
+    };
+
+    if c_ret == ffi::TILEDB_OK {
+       Ok(())
+    } else {
+       Err(Error::from("TileDB stats were not successfully reset."))
+    }
+}
+
+pub fn stats_dump_str() -> TileDBResult<String> {
+    let mut stats_dump_c_str = std::ptr::null_mut::<std::os::raw::c_char>();
+    let res = unsafe {
+        ffi::tiledb_stats_dump_str(
+            &mut stats_dump_c_str as *mut *mut std::os::raw::c_char,
```

Comment:
Prefer to use the `std::ffi::c_char` alias here. They changed that at some recent-ish point so things are a bit inconsistent but we've mostly standardized on the `std::ffi::*` names.